### ISSUE TITLE
Greenskins T1 Epic Quest "Da Stick's da Fing" : Bugged at step 14

### ISSUE THREAD

**Issue by NateTheSnek** [WarEmu/WarBugs#20649]
**Expected behavior and actual behavior:**
"Walk over to Brootus the Shaman and give him a good thumpin' to refresh his memory about the plan to overthrow the leadership of the Broketoof tribe. To do this, target Brootus and use the Bashin' Stick in your inventory. Brootus stands only a few feet away to the south. 
Speak with Brootus once the deed is done."

**Steps to reproduce the problem:**
Complete the quest through Step 13, by speaking to Rokgar at the "Gorgor's Smash" camp in Chapter 4.
Walk up to Brootus Blacktoof by the fire.
Target Brootus.
Attempt to use the Bashin' Stick in your quest inventory.

**Testing Screenshots/Videos/Evidences (always needed):**

![image](https://user-images.githubusercontent.com/23266079/211621085-db760ae9-a26e-4c48-b7eb-d8b20cbfad28.png)

**Comment by lilimx** [comment]
I cropped the screenshot

**Comment by Rubyeclipse-war** [comment]
Hello - thanks for the ticket!

I tested this and was able to complete this step properly. Could you try abandoning the quest, re-taking the latest step and giving this another try while targeting Brootus?

**Comment by lilimx** [comment]
Hey,

I am closing this ticket now as we have not received any further information regarding the problem.
If you are the opinion that the issue still persists, please open a new ticket with proper evidences.
